wiki/Beatmap/id.md

```diff
@@ -1,12 +1,118 @@
 ---
-outdated: true
-outdated_since: f4d914b0b826e19e50caeea90f58557065e9e0b7
+tags:
+  - mapset
+  - beatmapset
 ---
 
 # Beatmap
 
-*Artikel ini tidak mencakup seputar [Tingkat Kesulitan Beatmap](/wiki/Beatmap/Difficulty) secara mendalam.*
+**Beatmap** (terkadang disebut sebagai *beatmapset*) adalah sebuah set yang memiliki level gim (atau [tingkat kesulitan](#difficulty)) yang terdiri dari berbagai [hit object](/wiki/Hit_object) yang hampir selalu mewakili suatu lagu. Selain mewakili suatu lagu, beatmap juga mencakup komponen lainnya, semua dikemas dalam bentuk arsip [berekstensi .osz](/wiki/osu!_File_Formats/Osz_(file_format)):
 
-**Beatmap** merupakan istilah yang merujuk pada level-level permainan yang tersedia untuk dimainkan. Tiap-tiap berkas beatmap mengandung berbagai informasi yang terkait dengan aspek-aspek yang hadir di dalam permainan seperti [*hit object*](/wiki/Hit_Objects), [*timing*](/wiki/Beatmap_Editor/Timing), metadata, pengaturan tingkat kesulitan *(difficulty setting)*, efek-efek visual, dan lain sebagainya.
+- berkas lagu itu sendiri, yang tersimpan dalam format MP3 atau Ogg.
+- [background/gambar latar belakang](wiki/Beatmap/Background), atau sebuah video, yang berperan sebagai playfield.
```

Comment:
```suggestion
- [Background/gambar latar belakang](wiki/Beatmap/Background), atau sebuah video, yang berperan sebagai playfield.
```suggestion

Reply:
please also fix the slash `/`, which should be added
```suggestion
- [Background/gambar latar belakang](/wiki/Beatmap/Background), atau sebuah video, yang berperan sebagai playfield.
```